_monitoring-plugins/ad/api.md

```diff
@@ -353,75 +370,164 @@ POST _plugins/_anomaly_detection/detectors/_validate/detector
-    "period": {
-      "interval": 1,
-      "unit": "Minutes"
-    }
+  "period": {
+    "interval": 1,
+    "unit": "Minutes"
+  }
   },
   "window_delay": {
-    "period": {
-      "interval": 1,
-      "unit": "Minutes"
-    }
+  "period": {
+    "interval": 1,
+    "unit": "Minutes"
+  }
   }
 }
 ```
 
-If the validate API doesn’t find any issue in the detector configuration, it returns an empty response:
+If the validate detector API doesn’t find any issue in the detector configuration, it returns an empty response:
 
 #### Sample response
 
 ```json
 {}
 ```
 
-If the validate API finds an issue, it returns a message explaining what's wrong with the configuration. In this example, the feature query aggregates over a field that doesn’t exist in the data source:
+If the validate detector API finds an issue, it returns a message explaining what's wrong with the configuration. In this example, the feature query aggregates over a field that doesn’t exist in the data source:
 
 #### Sample response
 
 ```json
 {
   "detector": {
-    "feature_attributes": {
-      "message": "Feature has invalid query returning empty aggregated data: average_total_rev",
-      "sub_issues": {
-        "average_total_rev": "Feature has invalid query returning empty aggregated data"
+  "feature_attributes": {
+    "message": "Feature has invalid query returning empty aggregated data: average_total_rev",
+    "sub_issues": {
+    "average_total_rev": "Feature has invalid query returning empty aggregated data"
+    }
+  }
+  }
+}
+```
+
+The following request validates against the source data to see if model training might succeed. In this example, the data is ingested at a rate of every 5 minutes, and detector interval is set to 1 minute.
+
+```json
+POST _plugins/_anomaly_detection/detectors/_validate/model
+{
+  "name": "test-detector",
+  "description": "Test detector",
+  "time_field": "timestamp",
+  "indices": [
+  "server_log*"
+  ],
+  "feature_attributes": [
+  {
+    "feature_name": "test",
+    "feature_enabled": true,
+    "aggregation_query": {
+    "test": {
+      "sum": {
+      "field": "value"
+      }
+    }
+    }
+  }
+  ],
+  "filter_query": {
+  "bool": {
+    "filter": [
+    {
+      "range": {
+      "value": {
+        "gt": 1
+      }
+      }
+    }
+    ],
+    "adjust_pure_negative": true,
+    "boost": 1
+  }
+  },
+  "detection_interval": {
+  "period": {
+    "interval": 1,
+    "unit": "Minutes"
+  }
+  },
+  "window_delay": {
+  "period": {
+    "interval": 1,
+    "unit": "Minutes"
+  }
+  }
+}
+```
+
+If the validate detector API finds areas of improvement with your configuration, it returns a response with suggestions about how you can change your configuration to improve model training chances.
+
+#### Sample Responses
+
+In this example, the validate detector API return a responses with indicating that changing the detector interval length to at least 4 minutes can increase the chances of successful model training.
```

Comment:
Missed this on quip doc but should be: "In this example, the validate detector API **returns a response indicating** that changing..."